docs/symfony-integration.md

```diff
@@ -79,161 +105,168 @@ services:
 
-We also need reload both Zipkin and Jaeger on our browser, using the URLs `http://localhost:9411/`  and `http://localhost:16686/`. Do ensure that both your symfony server and docker instance are running for this step. 
+We also need reload both Zipkin and Jaeger on our browser, using the URLs `http://localhost:9411/`
+and `http://localhost:16686/`. Do ensure that both your symfony server and docker instance are running for this step.
 
-For Jaeger under service, you should see a `Hello World Web Server Jaeger` service, go ahead and click find traces to see exported traces.
+For Jaeger under service, you should see a `Hello World Web Server Jaeger` service, go ahead and click find traces to
+see exported traces.
 
 )
 ![image](https://user-images.githubusercontent.com/22311928/115637071-443ed880-a307-11eb-8521-78c803dcd623.png)
 
-
 Once we click on `Find Traces` you should be able to see traces like below:
 
 ![image](https://user-images.githubusercontent.com/22311928/115637073-44d76f00-a307-11eb-9e80-aa8f152577a8.png)
 
-
-We can click on a trace to get more information about the trace. 
+We can click on a trace to get more information about the trace.
 
 ![image](https://user-images.githubusercontent.com/22311928/115637075-44d76f00-a307-11eb-8d8e-5bf559bd2bf6.png)
 
-
-For Zipkin, we can visualize our trace by clicking on `Run Query` 
-
+For Zipkin, we can visualize our trace by clicking on `Run Query`
 
 ![image](https://user-images.githubusercontent.com/22311928/115637078-45700580-a307-11eb-9867-aa0c1a69f9fe.png)
 
-Since resources in Symfony's `public\index.php` file are available to the entire application, we can use any of the already instantiated tracers within `HelloController`. In addition to the tracers, we can also utilize associated properties, methods and events.
+Since resources in Symfony's `public\index.php` file are available to the entire application, we can use any of the
+already instantiated tracers within `HelloController`. In addition to the tracers, we can also utilize associated
+properties, methods and events.
 
-Lets try using the `addEvent` method, to capture errors within our controller as follows:
+Let's try using the `addEvent` method, to capture errors within our controller as follows:
 
 ```php
 global $zipkinTracer;
-        if ($zipkinTracer) {
-            /** @var Span $span */
-            $span = $zipkinTracer->getActiveSpan();
-            
-            $span->setAttribute('foo', 'bar');
-            $span->updateName('New name');
-
-            $zipkinTracer->startAndActivateSpan('Child span');
-            try {
-                throw new \Exception('Exception Example');
-            } catch (\Exception $exception) {
-                $span->setSpanStatus($exception->getCode(), $exception->getMessage());
-            }
-            $span->end();
-        }
+if ($zipkinTracer) {
+    /** @var Span $span */
+    $span = $zipkinTracer->getActiveSpan();
```

Comment:
`Tracer::getActiveSpan();` no longer exists. How will we achieve the equivalent to this now?

Reply:
`AbstractSpan::getCurrent()` to get the active span.

Reply:
Thank you